java/spacewalk-java.spec

```diff
@@ -530,14 +533,18 @@ install -d -m 755 $RPM_BUILD_ROOT%{cobdirsnippets}
 install -d -m 755 $RPM_BUILD_ROOT/%{_localstatedir}/lib/spacewalk/scc
 install -d -m 755 $RPM_BUILD_ROOT/%{_localstatedir}/lib/spacewalk/subscription-matcher
 
+mkdir -p $RPM_BUILD_ROOT%{shareddir}/susemanager/salt/salt_ssh/temp_bootstrap_keys
+mkdir -p $RPM_BUILD_ROOT%{shareddir}/susemanager/pillar_data/images
```

Comment:
`/srv/susemanager/pillar_data/images` is no longer used in 4.3.
Directory itself was not removed because of support for older images - there is no automatic migration from pre-4.3 images to 4.3 images and db based pillars.

From saltboot POV, we only need that salt is able to read the directory for pillar data if there is anything from 4.2 era. We no longer write there anything.

So it's ok to move it somewhere else or even remove it if we make a migration note that 4.2 images are not migrated and needs to be rebuild under 4.3 or 5.0.